lib/pliny/log.rb

```diff
@@ -72,6 +72,17 @@ def stderr
       @stderr
     end
 
+    def log_scrubber=(s)
```

Comment:
I'm sorry I missed this yesterday, but can you rename the `s` parameter to `scrubber`?